region/crateria/central.json

```diff
@@ -3195,6 +3195,19 @@
                   "name": "Base",
                   "notable": false,
                   "requires": [ "Morph" ]
+                },
+                {
+                  "name": "G-mode Morph to Powerbomb Bomb Blocks",
```

Comment:
Is this strat possible? I thought the PB won't break the bomb blocks until exiting gmode, and then you'll need a way to get through the morph tunnel. But if you have PB+Morph you can do it the vanilla way